Library/Homebrew/cmd/update.sh

```diff
@@ -24,7 +24,12 @@ git() {
 
 git_init_if_necessary() {
   BREW_OFFICIAL_REMOTE="https://github.com/Homebrew/brew"
-  CORE_OFFICIAL_REMOTE="https://github.com/Homebrew/homebrew-core"
+  if [[ -n "$HOMEBREW_MACOS" ]] || [[ -n "$HOMEBREW_FORCE_HOMEBREW_ORG" ]]
```

Comment:
This is just used by `brew test-bot`, right?

Reply:
`HOMEBREW_FORCE_HOMEBREW_ORG` you mean? Yes, I believe so.
See https://github.com/Homebrew/homebrew-test-bot/blob/ac511e47217a49dfbeffcb2f09e384f8aa49e13b/cmd/brew-test-bot.rb#L1325-L1329